agenta-web/package.json

```diff
@@ -40,6 +41,7 @@
         "postcss": "8.4.23",
         "posthog-js": "^1.77.2",
         "react": "18.2.0",
+        "react-ace": "^10.1.0",
```

Comment:
https://github.com/securingsincity/react-ace ? they stopped maintaing it no ?

Reply:
Yes, true.